modules/core/src/main/java/org/apache/ignite/internal/jdbc2/JdbcConnection.java

```diff
@@ -100,6 +102,15 @@
     /** Null stub. */
     private static final String NULL = "null";
 
+    /** Multiple statements supported since version. */
+    private static final IgniteProductVersion MULTIPLE_STATEMENTS_SUPPORTED_SINCE =
+        IgniteProductVersion.fromString("2.4.0");
+
+    /** Multiple statements V2 task supported since version. */
+    private static final IgniteProductVersion MULTIPLE_STATEMENTS_TASK_V2_SUPPORTED_SINCE =
+        IgniteProductVersion.fromString("2.8.0");
+
```

Comment:
I guess there should be 1 empty line. In Apache TC checkstyle disallows to have multiple empty lines.